content/engineering/bruteforce-lock-and-unlock/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Brute force lockout implementation"
+date: "2024-05-08"
+coverImage: ""
+author: "Gayathri Suresh"
+tags: ["Engineering","API","Identity Experience Framework", "Login", "Authentication", "Brute force lockout"]
+description: "In this blog, we'll learn about brute force lockout, creation of basic app using Identity Experience Framework, and on how to unlock an user account using APIs."
+---
+
+
+# Brute force lockout and unlock
+
+---
+
+## Contents
+- [Basic Terminologies](#basic-terminologies)
+- [API Implementation](#api-implementation)
+    - [Creating a Basic Application](#creating-a-basic-application)
+    - [Brute force lockout implementation](#brute-force-lockout-implementation)
+    - [Unlocking an account locked through brute force lockout](#unlocking-an-account-locked-through-brute-force-lockout)
+        - [Account Update API from the LoginRadius Account API collections](#account-update-api-from-the-loginradius-account-api-collections)
+        - [Auth Unlock Account by Access Token from the LoginRadius Authentication API collections](#auth-unlock-account-by-access-token-from-the-loginradius-authentication-api-collections)
+- [Conclusion](#conclusion)
+- [Miscellaneous](#miscellaneous)
+
+---
+
+## Basic Terminologies
+**Brute force attack:**
+- A method where every possible combination of characters or values is systematically tried to gain unauthorized access to a system, application, or data.
+
+**Brute force lock:**
+- Brute force lock is a type of account lock made to prevent a bruteforce attack.
+
+**Brute force lockout:**
+- Brute force lockout is a security mechanism that blocks access after a certain number of failed authentication attempts to prevent unauthorized access through repeated trial and error.
+
+**Multi-Factor Authentication (MFA):**
+-  MFA is a security method that requires multiple forms of identification to grant access, typically combining something the user knows, has, and is.
+
+In _LoginRadius_, the brute force lockout can be implemented in two ways, using:
+1. APIs
+2. Workflows
+
+> To implement brute force lockout in either of the above-mentioned ways, the user should own an admin account in the _LoginRadius Admin Console_.
+
+In this blog, we'll go through the API implementation of brute force lockout and user unlock
+
+---
+
+## API implementation
+### Creating a Basic Application
+1. In order to achieve brute force lockout using API, we need create a simple app with the login and registration features.
+2. This can be done by using the _Admin Console_.
+3. Navigate through ***Deployment->Identity Experience Framework***.
+4. Using theme, customization, preview, implement options, we can design the required application.
+5. If additional customization is required, the user can customize the predefined templates using the customize option, and then configure it as per the requirement.
```

Comment:
Did not able to understand this sentence, the user might be confused here because this is not mandatory steps as LR provide predefined theme right